source/includes/_cancel.md

```diff
@@ -0,0 +1,642 @@
+# Cancel
+> **Definition**
+
+```
+POST https://demo.ticketswitch.com/f13/cancel.v1?transaction_uuid={trans_uuid}&cancel_items_list=1,2
+```
+
+This endpoint allows users to attempt to cancel [purchased](#purchase) transactions (there is no guarantee that cancellations will succeed).
+By default, users are unable to cancel transactions.
+If you believe that you should be able to cancel transactions then contact us api@ingresso.co.uk.
+
+If you would like to attempt to cancel a transaction, follow these steps:
+1. Check if the order items within the transaction are cancellable by looking at the `cancellation_status` of each of the order items in the [purchase](#purchase) or [status](#status) response (they are cancellable if `cancellation_status` is `possible`).
+2. Attempt the cancellation of the `"cancellation_status": "possible"` order items. You can specify which order items to cancel in the transaction with the optional list, `cancel_items_list`. If you don't specify any order items to cancel then it will be assumed that the whole transaction should be cancelled.
```

Comment:
If some of the items have cancellation_status `possible` and some don't, then calling `cancel.v1` without `cancel_items_list` cancels all the ones it can?

Reply:
yes, I'll update it to make that clearer.